docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
How about if, in the paragraph above, we expand it to say "PyQt5, PyQt4 or PySide". The install line is introduced with "e.g.", which means that it's an example, so that seems pretty clear to me.